packages/@expo/metro-config/src/transform-plugins/__tests__/import-export-plugin-snapshots.test.ts

```diff
@@ -33,6 +33,10 @@ describe.each([
     test('re-export all')`
```

Comment:
Can we add an `exports default` case at the top. I forgot to add that.

```js
const local = 'local';
export default local;
```

And potentially a few disconnected import + export cases, e.g.

```js
import * as Test from 'test';
test(Test);
const local = 'local';
export default local;
```

and

```js
import Test from 'test';
test(Test);
const local = 'local';
export default local;
```

and

```js
import { Test } from 'test';
test(Test);
const local = 'local';
export default local;
```

The reason is that `export default` here potentially is a special case that requires a live binding at this "leaf export"